google_places.py

```diff
@@ -61,8 +61,8 @@ def change_location_query(address, key):
         address, key)
     r = requests.get(url)
     x = r.json()
+    print(x)
```

Comment:
why this change?

Reply:
I was checking the response returned by API as it was giving error that you have to add the billing account to continue using it. Should I remove it?

Reply:
yes, remove it

Reply:
Removed the comment also.